packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -54,9 +60,8 @@ export const useAuthModal = props => {
     }, [showMyAccount]);
 
     const handleSignOut = useCallback(() => {
-        // TODO: Get history from router context when implemented.
-        signOut({ history: window.history });
-    }, [signOut]);
+        signOut({ history, revokeToken });
```

Comment:
Should we be calling resetStore here?

Reply:
It may be worth rewriting this to something like 

```js
await signOut({ revokeToken });
await apolloClient.resetStore();
refresh({ history }); // from router-helpers or window?
```

Reply:
Went ahead and did this in 1f680c4; I like the idea of not having any effects in actions.